ext/ffi/ffi.stub.php

```diff
@@ -33,10 +33,16 @@ public static function arrayType(FFI\CType $type, array $dimensions): ?FFI\CType
     /** @prefer-ref $ptr */
     public static function addr(FFI\CData $ptr): FFI\CData {}
 
-    /** @prefer-ref $ptr */
-    public static function sizeof(object $ptr): ?int {}
+    /**
+     * @param FFI\CData|FFI\CType $ptr
+     * @prefer-ref $ptr
+     */
+    public static function sizeof($ptr): ?int {}
 
-    /** @prefer-ref $ptr */
+    /**
+     * @param FFI\CData|FFI\CType $ptr
+     * @prefer-ref $ptr
+     */
     public static function alignof(object $ptr): ?int {}
```

Comment:
It seems to me that this methods cannot return `null`; the [return](https://github.com/php/php-src/blob/959d3c9636fd070c554b281da221899638e2fe99/ext/ffi/ffi.c#L4193-L4194) should likely be `RETURN_THROWS()`. Same for `::sizeof()`.

Reply:
Uh, yes.

Reply:
Adjust the `return;` while here?

Reply:
Ah, sure... I had such plans at some point :D